clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/validate_json.py

```diff
@@ -101,3 +101,29 @@
 
     print(yaml.dump(temp_dict, default_flow_style=False).replace("u'", "'"))
     sys.exit(1)
+
+
+# Carry out final check, that no duplicated keys are present.
+# This check should be done last, after we have confirmed the json is valid.
+def fail_if_duplicated_keys(json_info):
```

Comment:
It is probably worth commenting what `json_info` is - I doubt the reader will know and not knowing makes the code hard to understand.

Reply:
Good point. I've done this - would you mind having another look to confirm my understanding is correct?